src/index.ts

```diff
@@ -200,7 +200,7 @@ async function app() {
       console.log(`filtering dependencies for ${JSON.stringify(String(dependencyList).split(','), null, 2)}\n`)
 
     }
-    writeToCSV(`org-slug,org-id,dep-id,dep-name,dep-version,project-name,project-id,project-url`)
+    writeToCSV(`org-slug,org-id,dep-id,dep-name,dep-version,latestVersion,latestVersionPublishedDate,firstPublishedDate,isDeprecated,project-name,project-id,project-url`)
```

Comment:
we'll want to match the convention of the header values here, e.g. `latest-version` instead of `latestVersion` etc..

Reply:
sounds good (just had copied keys from json output)